pkg/daemon/ceph/osd/volume.go

```diff
@@ -582,9 +582,15 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		}
 		mdArgs = append(mdArgs, strings.Split(conf["devices"], " ")...)
 
+		// Do not change devices passed by-id
```

Comment:
It's not limited to by-id. "Do not change device names if udev persistent names are passed" or so is more suitable.